src-cpp/app/modules/main/module.cpp

```diff
@@ -33,6 +37,11 @@ void Module::load()
     m_controllerPtr->init();
     m_viewPtr->load();
 
+    // Add Wallet Section to Sections model
+    auto walletSectionItem = new SectionItem(WALLET_SECTION_ID, SectionType::Wallet, WALLET_SECTION_NAME, "", "", WALLET_SECTION_ICON, "", false);
```

Comment:
Should we use `make_unique` instead of `new` to not use raw pointers? If not, is it possible we could be running into a memory leak here due to not deleting the walletSectionItem later? (which I think should happen if we logout)

Reply:
yes that's a miss from me!! will fix  thanks for pointing it out

Reply:
Done! please do check , awaiting approvals @richard-ramos , @alaibe

Reply:
@richard-ramos are you sure about `unique` here?

Reply:
hey @borismelnik Iv passed it as a shared_ptr now. directly in the view.